maps/away/crystalized_drugs/crystalized_drugs.dm

```diff
@@ -0,0 +1,93 @@
+#include "crystalized_drugs_areas.dm"
+#include "crystalized_drugs_items.dm"
+#include "crystalized_drugs_tiles.dm"
+
+/datum/map_template/ruin/away_site/crystaldrugs
+	name = "Drug Manufacturing Lab"
+	id = "awaysite_drug_lab"
+	description = "An active drug manufacturing facility with smuggler ship."
+	suffixes = list("crystalized_drugs/crystalized_drugs-1.dmm")
+	spawn_cost = 1
+	player_cost = 0
+	generate_mining_by_z = 1
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/bacchus)
+
```

Comment:
should probably add this for the weekend so people can mess with the new map. i did that for the destroyed colony redux, and we've done that in the past for all new awaymaps. I'll remove it after the weekend's done.
```suggestion
	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
	
```